src/app/views.py

```diff
@@ -652,7 +653,7 @@ def check_license(request):
     else:
         return HttpResponseRedirect(settings.LOGIN_URL)
 
-
+@csrf_exempt
```

Comment:
This can be risky for normal usecase. Is there a way to bypass csrf in any other way? Or pass dummy csrf from  the page?

Reply:
Yes, we can try, I had not thought of this approach, will try this once and let you know once done

Reply:
Hi @rtgdk , I have resolved the above issue by removing the `csrf_exempt` and instead passing the `var csrftoken = $('input[name=csrfmiddlewaretoken]').val();` as a header for our `ajax` request, so now it is also working correctly